GSQ Sampling Method Vocabulary

KellyVance opened the pull request:
Tech Review @LukeHauck or @DavidCrosswellGSQ 
Vocabulary Owner and SME @SalEdwards01 
FYI- @lisa-woody 

This vocabulary will be ready to use when we need it, but for now I strongly recommend we stay the course until after deployment and let SRA continue to use the GA vocabulary directly. At least for now where every change impacts the delivery schedule. 

This vocabulary links directly to the GA vocabulary in full, with the addition of Unconsolidated Sediment Sampling as per recommendation by @SalEdwards01 and @MicaelaPreda 

I have also explanded the definitions for still and video photography as recommended to explicitly state that it is platform independent (i.e. can be handheld, fixed, drone, etc)

LukeHauck commented on vocabularies/sampling-method.ttl:
```diff
@@ -0,0 +1,890 @@
+    sdo:url <https://www.business.qld.gov.au/industries/mining-energy-water/resources/geoscience-information/gsq> .
+
+smpmh:unconsolidated_sediment_sampling a skos:Concept ;
+	dcterms:created "2020-06-03"^^xsd:date ;
```
Turtle validator giving me **Undefined prefix "dcterms:" on line 48.**

KellyVance replied:
Good pick-up. Changed all dcterms to dct.

KellyVance commented:
> I'm assuming this is back to the GA version with the 'unconsolidated sediment sampling' method added, as the comments I made regarding use of drones for still and video photography are not present.
> 
> All good. Luke picked up the line 48 messsage.

@SalEdwards01 
Unconsolidated Sediment Sampling has been added
Still Photography (lines 200-206) i've changed the defintion
```
gasmp:still_photography a skos:Concept ;
	dct:created "2018-01-30T23:26:29Z"^^xsd:dateTime ;
	dct:creator <http://editor.vocabs.ands.org.au/user/GA_Sedgmen.Lara_Admin> ;
	dct:source "Geoscience Australia"@en ;
	skos:definition "A still image captured by a camera. Platform independent, i.e. image may be captured via a handheld camera, drone, camera trap, fixed camera."@en ;
	skos:prefLabel "still photography"@en ;
	skos:topConceptOf <http://linked.data.gov.au/def/sampling-method> .
```

video_photography (lines 239-245) i've changed the defintion

```
gasmp:video_photography a skos:Concept ;
	dct:created "2018-01-30T23:26:29Z"^^xsd:dateTime ;
	dct:creator <http://editor.vocabs.ands.org.au/user/GA_Sedgmen.Lara_Admin> ;
	dct:source "Geoscience Australia"@en ;
	skos:definition "Moving images captured by a video camera, either digital or film. Platform independent, i.e. image may be captured via a handheld camera, drone, camera trap, fixed camera."@en ;
	skos:prefLabel "video photography"@en ;
	skos:topConceptOf <http://linked.data.gov.au/def/sampling-method> .
```
